fsharp-backend/tests/Tests/Canvas.Tests.fs

```diff
@@ -79,17 +91,29 @@ let testHttpLoadIgnoresDeletedFns =
     do!
       Canvas.saveTLIDs
         meta
-        [ (handler.tlid, [ hop handler ], PT.TLHandler handler, Canvas.NotDeleted)
-          (f.tlid, [ PT.SetFunction f ], PT.TLFunction f, Canvas.NotDeleted) ]
+        [ (handler.tlid,
+           [ hop handler ],
+           PT.Toplevel.TLHandler handler,
+           Canvas.NotDeleted)
+          (f.tlid, [ PT.SetFunction f ], PT.Toplevel.TLFunction f, Canvas.NotDeleted) ]
```

Comment:
I find myself asking "... what's 'f' here?". Really easy to lose context in a PR review. Could we rename?

Reply:
I don't disagree, but I just hit this from a module rename and didn't touch it otherwise. Would rather not increase scope on this.